docs/run/runtime-options-overview.mdx

```diff
@@ -0,0 +1,559 @@
+| resilience_level     |                          |                         | `0`/`1`/`2`                                                                                     | `1`                         |
+| seed_estimator       |                          |                         |  integer                                                                                |     `None`               |
+| simulator            |                          |                         |                                                                                   |                     |
+|                      | noise_model              |                         |   [Qiskit Aer NoiseModel](../verify/building_noise_models), or its representation        | `None`                      |
+|                      | seed_simulator           |                         |   integer                                                                               | `None`                    |
+|                      | coupling_map             |                         |  List where each entry specifies a directed two-qubit interaction, for example: [[0, 1], [0, 3], [1, 2], [1, 5], [2, 5], [4, 1], [5, 3]]     | full connectivity                   |
+|                      | basis_gates              |                         |  Set of single- and two-qubit gates that provide a universal gates set, meaning that any quantum operation can be decomposed into those gates. Example: `['ecr', 'x', 'sx', 'rz']` | The set of all basis gates supported by [Qiskit Aer simulator](https://qiskit.github.io/qiskit-aer/stubs/qiskit_aer.AerSimulator.html) |
+| transpilation        | skip_transpilation       |                         | `True`/`False`                                                                                      | `False`                     |
+|                      | initial_layout           |                         |                                                                                         |                         |
+| twirling             | enable_gates             |                         | `True`/`False`                                                                                      | `False`                     |
+|                      | enable_measure           |                         | `True`/`False`                                                                                      | `True`                      |
+|                      | num_randomizations       |                         |  `auto`/ integer >= 1 (Note that this value strongly influences execution time.)            | `'auto'`                    |
+|                      | shots_per_randomization  |                         |   `auto`/ integer >= 1.  The maximum depends on the maximum shots allowed by the system. To find this value, run `backend.max_shots`.                                                                      | `'auto'`                    |
```

Comment:
The description seems wordier than the rest. Maybe just "`auto` / integer >= 1 and <= `backend.max_shots`"